app/frontend/wallet/cardano-ledger-crypto-provider.js

```diff
@@ -0,0 +1,124 @@
+const TransportU2F = require('@ledgerhq/hw-transport-u2f').default // for browser
+const Ledger = require('ledgerhq/hw-app-ada').default // temporary hack, should be @ledgerhq
+const CachedDeriveXpubFactory = require('./helpers/CachedDeriveXpubFactory')
+const cbor = require('borc')
+const {TxWitness, SignedTransactionStructured} = require('./transaction')
+
+const CardanoLedgerCryptoProvider = async (ADALITE_CONFIG, walletState) => {
+  const transport = await TransportU2F.create()
+  transport.setExchangeTimeout(15 * 60000) // 60000ms = 1minute
```

Comment:
move to constants?

Reply:
I think this is Ledger specific, but perhaps the "LOGOUT_AFTER" constant could be reused, which makes sense to me.